lib/membrane_live_web/controllers/webinar_controller.ex

```diff
@@ -74,8 +74,11 @@ defmodule MembraneLiveWeb.WebinarController do
     end
   end
 
-  defp is_user_authorized(webinar, jwt_user_uuid) when jwt_user_uuid == webinar.moderator_id,
-    do: {:ok, webinar}
+  defp is_user_authorized(webinar, jwt_user_uuid, _callback)
+       when jwt_user_uuid == webinar.moderator_id,
+       do: {:ok, webinar}
 
-  defp is_user_authorized(_webinar, _jwt_user_uuid), do: {:error, :forbidden}
+  defp is_user_authorized(webinar, _jwt_user_uuid, callback) do
+    if callback == (&show_callback/2), do: {:ok, webinar}, else: {:error, :forbidden}
+  end
```

Comment:
And because change mentioned above it could look like this:
```suggestion
  defp is_user_authorized(webinar, jwt_user_uuid, true), do {:ok, webinar}
  defp is_user_authorized(webinar, jwt_user_uuid, _is_show_callback?)
       when jwt_user_uuid == webinar.moderator_id,
       do: {:ok, webinar}

  defp is_user_authorized(webinar, _jwt_user_uuid, _is_show_callback?), do: {:error, :forbidden}
```